docs/java/how-to-develop-a-workflow-definition-in-java.md

```diff
@@ -41,23 +40,3 @@ A Workflow implementation implements a Workflow interface.
     }
   }
 ```
-
-To call Activities in your Workflow, call the Activity implementation.
```

Comment:
Not sure if we want to remove this before attempting to rewrite the java narrative vertically.